### DISCUSSION TITLE
One more question - NES dumping

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#707] [Need help]
After dumping about 25 games without an issue - some requiring me to select a letter, then the ROM, I ran into two games in a row, that both showed up as Duck Tails (USA) (Aftermarket) (Homebrew).nes - I cleaned them extremely well repeatedly and get the same results every time. Odd that it was two in a row after many others either worked, or prompted me to pick manually. The two here are Pro Wrestling and Tetris 2.

After manually selecting them - which takes forever and has my wrist hurting (am I missing a faster way), the rom matches up fine, and the resulting NES file matched the CRC32 for known ones online. It even seemed to auto-pick Rev A for Pro Wrestling even though I selected the original, as I didn't know. If I leave it on the auto selected Ducktails, it says not found on the CRC32 check.

The main issue here is that I don't see a way to select a letter, and scrolling from D to T was quite tedious. Also, why two games both jumping to this one, but dumping perfect ROMs? I've not dumped any Ducktails game at any point so far.

**Comment by sanni** [comment]
That happens if there is an error in the nes.txt database. Could you post the log files found in the same directory as the wrongly detected ROMs? 

Example:
![wgfafsadffafafff](https://user-images.githubusercontent.com/19169494/218257084-35d76fac-3408-4637-b0b9-c686f588a950.jpg)

**Comment by PsychoFox11** [comment]
Sure, here ya go! Still very odd that it was 2 in a row, and none before or since have done it, but those two did! Lucky I guess!
They don't appear to have anything after the 'searching database' line for a long time - lots of white space before the rest - is that expected, or another issue? Perhaps one line break for every title I scrolled by? lol

[ProWrestling.txt](https://github.com/sanni/cartreader/files/10717777/ProWrestling.txt)
[Tetris2.txt](https://github.com/sanni/cartreader/files/10717779/Tetris2.txt)

**Comment by sanni** [comment]
Yep, was a line break for every scrolled by title. Oops. Fixed now. 

Also corrected the database entries for your wrongly detected games. Can you download the new [nes.txt](https://github.com/sanni/cartreader/archive/refs/heads/master.zip), copy it to your SD card and verify that the games are now detected?

**Reply by PsychoFox11** [reply]
Yep, tested both and they showed up properly! Thanks. Only 2 out of the ~60 or so NES games I've done so far did that.

**Comment by sanni** [comment]
Two more notes:

- In the NES database browser you can enable fast scrolling when you press and hold the rotary encoder button until the LED lights up. It scrolls 30 entries at once, to deactivate hold the button pressed until the LED lights up again.

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/ihO57j_eyWU/0.jpg)](https://www.youtube.com/watch?v=ihO57j_eyWU)      

- If you want you can post dump logs of any game where you had to manually select the title (besides region or revision) and I can try to fix their database entry.
